starter_code/views/album.ejs

```diff
@@ -0,0 +1,12 @@
+<a href="/"><button>BackHome</button></a>
+<div class>
+        <% albums.forEach((album) => { %>
+        <div>
+            <h3><%= album.name %></h3>
+            <% if (album.images.length > 0) { %>
```

Comment:
Good call! However, the best approach is define a helper function that, if there is an image, return it, otherwise displays a placeholder. Then you can use that function in artists.ejs too.